ArduCopter/autoyaw.cpp

```diff
@@ -264,3 +270,30 @@ float Mode::AutoYaw::rate_cds() const
     // return zero turn rate (this should never happen)
     return 0.0f;
 }
+
+void Mode::AutoYaw::update_weathervane(const int16_t pilot_yaw, const int16_t roll_cdeg, const int16_t pitch_cdeg)
```

Comment:
normally the order of inputs is roll, pitch, yaw...

Reply:
changed the ordering to match rpy.  yaw can actually be an angle, rate, or rcin.  So long as it represents a pilot's request.